rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -61,9 +61,10 @@ In addition to the registered header parameters, the following headers MUST be p
 
 * **sigt**: \(signing time\) MUST contain the signing time of the transaction as Unix time since epoch encoded as NumericValue.
 * **ver**: MUST contain the format version of the transaction as number. For this version of the format the version MUST be 1.
-* **prevs**: \(previous transactions\) MUST contain the references \(see section 3.2\) of the preceding transactions \(see section 3.4\).
+* **prevs**: \(previous transactions\) MUST contain references \(see section 3.2\) of preceding transactions \(see section 3.4\).
   * When it's a root transaction the field SHALL NOT have any entries.
   * When creating a transaction it MUST only contain transactions that the local node has successfully processed, to avoid publishing unprocessable transactions.
+  * The field SHOULD contain only one of the latest heads in addition to other requirements (for example requirements introduced by [RFC006 §5.1](rfc006-distributed-registry#51-conflict-detection)).
```

Comment:
In RFC016 we state that the LC is calculated as 'prev with highest LC + 1'. The current formulation allows using any of the current heads. 

This might cause a conflict in LC calculation. If we only add 1 of the heads to the prevs, the list of heads will eventually grow out of control. 

* The field SHOULD contain at most 5 references 
* One of the references MUST be the head with the highest LC. If there are multiple heads with the same highest LC, at least one MUST be added as a reference.

One is going to be a reference to the Tx containing the key used for signing. That currently leaves 3 prevs that can be chosen freely and may be assigned to specific references later on.

Reply:
it should not use the term 'head' but 'one of the transactions with the highest LC' There is no list of heads anymore so that won't be an issue.

Reply:
It might be better indeed to place RFC016 content inside RFC004